samples/integration/vcpkg/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+# VCPKG integration.
+if(DEFINED ENV{VCPKG_ROOT} AND NOT DEFINED CMAKE_TOOLCHAIN_FILE)
+  set(CMAKE_TOOLCHAIN_FILE "$ENV{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake"
+    CACHE STRING "")
+endif()
+if(DEFINED ENV{VCPKG_DEFAULT_TRIPLET} AND NOT DEFINED VCPKG_TARGET_TRIPLET)
+  set(VCPKG_TARGET_TRIPLET "$ENV{VCPKG_DEFAULT_TRIPLET}" CACHE STRING "")
+endif()
```

Comment:
This file is meant for users to copy/paste and use in their applications. We should try to simplify it as much as possible. Is this actually needed?

Reply:
The expected flow to test/play this samples is:

- Clone the c++ repo.
- Open a sample folder
- build/run

And that's it.  Without setting vcpkg path, toolchain or anything. That's as simplest as I am trying to make this.

Then, once it's building/running, people can start looking at the code for things to copy/paste.